hello_world/Android.mk

```diff
@@ -0,0 +1,25 @@
+################################################################################
+# Android optee-hello-world makefile                                           #
+################################################################################
+LOCAL_PATH := $(call my-dir)
+
+CFG_TEEC_PUBLIC_INCLUDE = $(LOCAL_PATH)/../optee_client/public
+
+################################################################################
+# Build hello world                                                            #
+################################################################################
+include $(CLEAR_VARS)
+LOCAL_CFLAGS += -DANDROID_BUILD
+LOCAL_CFLAGS += -Wall
+
+LOCAL_SRC_FILES += host/main.c
+
+LOCAL_C_INCLUDES := $(LOCAL_PATH)/ta/include \
+		$(CFG_TEEC_PUBLIC_INCLUDE) \
+
+LOCAL_SHARED_LIBRARIES := libteec
+LOCAL_MODULE := tee_helloworld
```

Comment:
@jforissier it is the same as in original `hello_world` (https://github.com/linaro-swg/hello_world/blob/master/Android.mk), I decided to copy-paste sources + makefiles with minimal amount of changes